### ISSUE TITLE
docker-entrypoint.sh cleans up user table on startup

### ISSUE THREAD

**Issue by InputObject2** [danielbrendel/hortusfox-web#65]
I think the issue might be this step:

```bash
# Run database migrations
echo "Running database migrations..."
php asatru migrate:fresh
```

When running the command to check if the admin user exists, we get nothing:

```bash
# before
root@hortusfox-58c6b9b76c-qkxdz:/var/www/html# export ADMIN_EMAIL="admin@example.com"
root@hortusfox-58c6b9b76c-qkxdz:/var/www/html# mysql -u "$DB_USERNAME" -p"$DB_PASSWORD" -h "$DB_HOST" -D "$DB_DATABASE" -N -s -e "SELECT COUNT(*) FROM users WHERE email='$ADMIN_EMAIL';"
1

# after running php asatru migrate:fresh
root@hortusfox-58c6b9b76c-qkxdz:/var/www/html# mysql -u "$DB_USERNAME" -p"$DB_PASSWORD" -h "$DB_HOST" -D "$DB_DATABASE" -N -s -e "SELECT COUNT(*) FROM users WHERE email='$ADMIN_EMAIL';"
0
```

Is there a way with asatru migrate to init the db and perform upgrades and not drop the tables?

**Comment by danielbrendel** [comment]
The following command performs an upgrade rather then a complete re-creation:
```
php asatru migrate:list
```

**However** as I see currently in, for instance, the UserModel_Migration the up() method drops the table before adding the columns. That's kinda contrary to the intended behavior of this command. Basically when running this command it checks the sha512 of a migration class and compares it to a list. If it differs then it calls the migrations up() method. So whenever a migration is altered (altering columns, etc.) it requires the up method to handle it accordingly. 

At this point the current up() methods don't act accordingly. I'll have to consider what's the best approach here. For now I am sure this is app related, not framework related, so I can implement an update to the hortusfox app that deals with that.

**Comment by InputObject2** [comment]
I'll switch it up in the dockerfile. 

Also a bit unrelated but I saw that you added all the environment variables to a .env file in the dockerfile, I don't think that's needed for the service to run so I'd probably get rid of that part to minimize complexity.

**Comment by danielbrendel** [comment]
I had to revert this for now, because it created further problems. I'll need to take some more time to improve on this.

**Comment by InputObject2** [comment]
This was solved by #83
